checkov/cloudformation/checks/resource/aws/LambdaServicePermission.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+
+from checkov.cloudformation.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class LambdaServicePermission(BaseResourceCheck):
+    def __init__(self) -> None:
+        name = "Ensure that AWS Lambda function permissions delegated to AWS services are limited by SourceArn or SourceAccount"
+        id = "CKV_AWS_293"
+        supported_resources = ["AWS::Lambda::Permission"]
+        categories = [CheckCategories.GENERAL_SECURITY, ]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        properties = conf.get('Properties')
+        if properties is not None:
```

Comment:
```suggestion
        if properties and isinstance(properties, dict):
```
let's make sure, ew deal with a dictionary otherwise we will have a problem.